tests/test_csv_generator.py

```diff
@@ -5,11 +5,12 @@ class TestIterCsv():
     def test_it_creates_csv_lines_from_lit_of_rows(self):
```

Comment:
Just spotted typo in the test name `lit` -> `list`.  I reckon we can live with it for now if you can't face waiting for Travis again.